src/System.Management.Automation/namespaces/FileSystemProvider.cs

```diff
@@ -1539,8 +1605,15 @@ protected override void InvokeDefaultAction(string path)
                 if (isDirectory)
                 {
                     DirectoryInfo directory = new DirectoryInfo(path);
+                    InodeTracker tracker = null;
+
+                    if (recurse && Context.MyInvocation.BoundParameters.ContainsKey("FollowSymlink"))
```

Comment:
We can not define explicitly mandatory sub-parameters 😕 That would be a useful feature.
So it is more safe to leave "recurse" here.

If `-Depth`  implies  `-Recurse` I believe we can do the same with `-FollowSymlink `.